.env

```
@@ -0,0 +1,12 @@
+# Environment variables
+DATABASE_URL="postgresql://postgres:Buckalew2025@localhost:5432/Buckalew?schema=public"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)